src/main/cliapp/package.json

```diff
@@ -9,6 +9,7 @@
     },
     "license": "MIT",
     "dependencies": {
+        "@babel/plugin-proposal-private-property-in-object": "7.21.0-placeholder-for-preset-env.2",
```

Comment:
`babel/plugin-proposal-private-property-in-object` must be imported at a production-grade version (I'd propose `^7.21.11`), instead of the development version used here.

Reply:
set in last commit